src/components/application_manager/test/mobile_message_handler_v1_test.cc

```diff
@@ -121,10 +121,10 @@ TEST(MobileMessageHandlerTestV1Test,
   message->set_json_message(data_json);
   message->set_connection_key(connection_key_p1);
 
-  RawMessage* ptr =
-      MobileMessageHandler::HandleOutgoingMessageProtocol(message);
+  std::shared_ptr<RawMessage> ptr(
```

Comment:
@SNiukalov Can't see any sharing here. What is the reason to create shared_ptr?

Reply:
@AByzhynar 
Fixed in:https://github.com/LuxoftSDL/sdl_core/pull/41/commits/be4a738c2392d04c4fd0cd9a2dee39ce76b94c6e